### PR TITLE
fix `--bug-report=rr`

### PR DESCRIPTION
**Pull request by simeonschaub**
This option is currently broken on nightly due to world age errors

### CODE REVIEW

**Review by vtjnash — APPROVED** [review]
(I assume this test might need to be x86+linux only though?)

**Comment by simeonschaub** [comment]
Ah, true!

**Comment by vtjnash** [comment]
CI failed

File: base/client.jl [hunk 1]
```diff
@@ -246,7 +246,7 @@ function exec_options(opts)
             # If we're doing a bug report, don't load anything else. We will
             # spawn a child in which to execute these options.
             let InteractiveUtils = load_InteractiveUtils()
-                InteractiveUtils.report_bug(arg)
+                invokelatest(InteractiveUtils.report_bug, arg)
```
Comment (maleadt) [inline base/client.jl:249]: `@invokelatest` is a bit cleaner

Comment (simeonschaub) [reply base/client.jl:249]: I tried to copy the style that was used in the `Distributed` case just below. I don't feel like this makes a huge difference in terms of readability in the single-arg case

**Comment by simeonschaub** [comment]
@keno Any ideas why this would fail on our CI environment? It worked for me locally even without the selinux shenanigans. Can you think of a better way to do a simple rr smokescreen test?

**Comment by simeonschaub** [comment]
I skipped the test for now so we can at least merge this fix

**Pull request merged**